packages/@aws-cdk/aws-apigateway/test/integ.usage-plan.multikey.expected.json

```diff
@@ -1,8 +1,7 @@
 {
   "Resources": {
     "myusageplan4B391740": {
-      "Type": "AWS::ApiGateway::UsagePlan",
-      "Properties": {}
+      "Type": "AWS::ApiGateway::UsagePlan"
```

Comment:
Curious: what's causing this change?

Possibly from a different change in master?

Reply:
Possibly, but not sure

Reply:
This is now gone, so likely went away with a patch elsewhere.